### PR TITLE
026 add branding

### PR DESCRIPTION
**Pull request by ashnabraham**
026 add branding

### CODE REVIEW

**Review by selina-mcg — APPROVED** [review]

**Review by selina-mcg — DISMISSED** [review]

File: .idea/.gitignore
Comment (shaunganley) [inline .idea/.gitignore:1]: remove and ignore this directory

File: package.json [hunk 1]
```diff
@@ -1,30 +1,28 @@
...
   "scripts": {
     "start": "tsc && node dist/app.js",
-
+ 
      "linter": "npx eslint src/ ",
     "dev": "nodemon src/app.ts",
-
-
+ 
+
```
Comment (shaunganley) [inline package.json:8]: no need for empty lines

File: views/parent.html
Comment (shaunganley) [inline views/parent.html:1]: I would call this header.html

**Review by shaunganley — APPROVED** [review]

File: .idea/.gitignore
Comment (ashnabraham) [reply .idea/.gitignore:1]: done

Comment (ashnabraham) [reply .idea/.gitignore:1]: what do I delete within the .gitignore?

**Review by jackmknowles — APPROVED** [review]
Looks great

**Review by shaunganley — APPROVED** [review]

**Pull request merged**